compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -164,6 +164,11 @@ pub(crate) unsafe fn create_module<'ll>(
             // See https://github.com/llvm/llvm-project/pull/118004
             target_data_layout = target_data_layout.replace("-i128:128", "");
         }
+        if sess.target.arch.starts_with("wasm32") {
+            // LLVM 20 updates the powerpc64 layout to correctly align 128 bit integers to 128 bit.
+            // See https://github.com/llvm/llvm-project/pull/118004
```

Comment:
Question: copy-pasta'd comment? This should be https://github.com/llvm/llvm-project/pull/119204 right?

Reply:
sigh, yep - thanks!

(our LLVM HEAD CI spotted another case too, so I fixed that as well)